rotkehlchen/data/import_utils.py

```diff
@@ -0,0 +1,283 @@
+                else:
+                    cur_data['trade_type'] = TradeType.SELL if row['Operation'] == 'Sell' else TradeType.BUY  # noqa:  E501
+                    if amount < 0:
+                        cur_data['from_asset'] = cur_asset
+                        cur_data['from_amount'] = AssetAmount(-amount)
+                    else:
+                        cur_data['to_asset'] = cur_asset
+                        cur_data['to_amount'] = amount
+            prepared_trade_dicts.append(cur_data)
+
+        def trades_key(t_dict: Dict) -> str:
+            return ';'.join(sorted([f'{x[0]}: {x[1]}' for x in t_dict.items()]))
+
+        grouped_trades_dicts = []
+        for _, grouped_trades in groupby(sorted(prepared_trade_dicts, key=trades_key), key=trades_key):  # noqa: E501
+            grouped_trades_dicts.append(list(grouped_trades))
+
+        unique_trade_dicts = []
+        for trades_dict_group in grouped_trades_dicts:
+            result_trade_dict = trades_dict_group[0]
+            for trade_dict in trades_dict_group[1:]:
+                result_trade_dict['to_amount'] += trade_dict['to_amount']
+                result_trade_dict['from_amount'] += trade_dict['from_amount']
+                if result_trade_dict['fee_amount'] is not None:
+                    result_trade_dict['fee_amount'] += trade_dict['fee_amount']
+            unique_trade_dicts.append(result_trade_dict)
+
+        trades = []
+        for trade_dict in unique_trade_dicts:
+            rate = FVal(trade_dict['to_amount']) / FVal(trade_dict['from_amount'])
+            trades.append(Trade(
+                timestamp=timestamp,
+                location=Location.BINANCE,
+                base_asset=trade_dict['to_asset'],
+                quote_asset=trade_dict['from_asset'],
+                trade_type=trade_dict['trade_type'],
+                amount=trade_dict['to_amount'],
+                rate=Price(rate),
+                fee=trade_dict['fee_amount'],
+                fee_currency=trade_dict['fee_asset'],
+                link='',
+            ))
+        return trades
+
+    @classmethod
+    def process_entries(cls, timestamp: Timestamp, data: List[BinanceCsvRow]) -> int:
+        trades = cls.process_trades(timestamp=timestamp, data=data)
+        cls.db.add_trades(trades)
+        return len(trades)
+
+
+class BinanceStakingEntry(BinanceMultipleEntry):
+    @classmethod
+    def are_entries(cls, requested_operations: List) -> bool:
+        return (
+            requested_operations == ['ETH 2.0 Staking', 'ETH 2.0 Staking']
+        )
+
+    @classmethod
+    def process_entries(cls, timestamp: Timestamp, data: List[BinanceCsvRow]) -> int:
+        amount = abs(data[0]['Change'])
+        asset = data[0]['Coin']
+
+        ledger_action = LedgerAction(
+            identifier=0,
+            timestamp=timestamp,
+            action_type=LedgerActionType.EXPENSE,
+            location=Location.BINANCE,
```

Comment:
I'm not sure but I believe staking is not an expense. What would be better @LefterisJP ?

Reply:
Changed to Trade.